packages/web-components/src/components/va-table/va-table-inner/va-table-inner.tsx

```diff
@@ -55,6 +80,47 @@ export class VaTableInner {
     }
   }
 
+  fireSort(e: Event) {
+    const target = e.currentTarget as HTMLElement;
+    const th = target.closest('th');
+    const sortdir = th.dataset.sortdir;
+    const index = th.dataset.rowindex;
+    this.sortTable.emit({ index, sortdir });    
+  }
+
+  handleKeyDown(e: KeyboardEvent) {
+    if (e.key === 'Enter') {
+      this.fireSort(e);
+    }
+  }
+
+  getSortIcon(index: number, row: number): HTMLVaIconElement {
+    let icon: HTMLVaIconElement = null;
+    // th is in header
+    if (this.sortable && row === 0) {
+      // we just performed a sort on this column
+      if (this.sortindex !== null && this.sortindex === index) {
+        const icon_name = this.sortdir == 'descending' ? 'arrow_upward' : 'arrow_downward';
+        icon = <va-icon icon={icon_name} size={3} />
+      // we did not just perform a sort on this column
+      } else {
+        icon = <va-icon icon="sort_arrow" size={3} />
+      }
+      return (
+        <span
+          role="button"
```

Comment:
I think [USWDS](https://designsystem.digital.gov/components/table/) is using a `button` with styling classes. Can we do that too here instead of using a `span`?

![Screenshot 2024-10-15 at 11 27 36 AM](https://github.com/user-attachments/assets/8ada7bc1-51b8-4d64-b176-cb4dc21639dd)

Reply:
@jamigibbs - I switched to button but had to style it since the styles from uswds diverge from ours here

Reply:
@department-of-veterans-affairs/platform-design-system-designers Are we aware that there is a design divergence between USWDS table and our Figma?

Reply:
@jamigibbs - I have keyed Dan into this and he will be working on a solution going forward